src/test/mock.rs

```diff
@@ -29,14 +29,10 @@ pub struct Mock {
     non_exhaustive: bool,
 }
 
-static NEXT_PORT: AtomicU16 = AtomicU16::new(15420);
-
 impl Mock {
     #[allow(clippy::new_without_default)]
     pub fn new() -> Self {
-        // TODO: need to reassign if the port has already been taken.
-        let port = NEXT_PORT.fetch_add(1, Ordering::Relaxed);
-        let addr = SocketAddr::from(([127, 0, 0, 1], port));
+        let addr = SocketAddr::from(([127, 0, 0, 1], 0));
```

Comment:
I suppose it won't work on Windows, zero port properly handled by linux stack

Reply:
It will work, Windows handles this in the same way.